recipes/qt/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+libX11-devel
+libXt-devel
+libXext-devel
+libXrender-devel
+gtk2-devel
+dbus-devel
+libXi-devel
+mesa-libGLU-devel
```

Comment:
I'm not sure if these are the right package names in CentOS 6 (which is the version used in conda-forge, but not the one I use, which is CentOS 5).

@gillins, did you verify them?